app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All visitors must be vaccinated")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
```

Comment:
Use method `get` to avoid KeyError
```suggestion
        elif visitor["vaccine"].get("expiration_date") < date.today():
```